rust_snuba/src/processors/generic_metrics.rs

```diff
@@ -240,6 +246,17 @@ where
     T: Parse + Serialize,
 {
     let payload_bytes = payload.payload().context("Expected payload")?;
+    let use_case: MessageUseCase = serde_json::from_slice(payload_bytes)?;
+    let killswitch_config = get_str_config("key");
```

Comment:
How do you configure multiple use case killswitches, e.g. how do I killswitch `custom` and `transactions`?

Reply:
We'll use the Snuba runtime config convention, which just passes in a list as a string, so a killswitch config would look like "[use_case_1, use_case_2]"

Reply:
This seems a little janky, but I'm guessing this is because we are getting these configs through python and redis is not really typed?